src/compositemark/boxplot.ts

```diff
@@ -336,11 +336,11 @@ function boxParams(spec: GenericUnitSpec<Encoding<string>, BOXPLOT | BoxPlotDef>
         // Add bin or timeUnit transform if applicable
         const bin = channelDef.bin;
         if (bin) {
-          const {field} = channelDef;
-          bins.push({bin, field, as: transformedField});
+          const {field: fieldName} = channelDef;
```

Comment:
why do you need this rename? it seems inferior

Reply:
Because otherwise, we are shadowing the `field` variable, which is an imported method.

Reply:
Do you have a suggestion for how we could handle this differently? I'm surprised that the shadowing hasn't been visible until now.

Reply:
How about we rename the `field` method to `vgField`?  

I think that's better because there is no concept of "fieldName" in either VG or VL as they are both called `field`.  That said, our field method converts VL field def into a field in Vega.